msprime/demography.py

```diff
@@ -3150,6 +3183,35 @@ def _as_lineage_movements(self):
         ]
 
 
+@dataclasses.dataclass
+class ActivatePopulationEvent(DemographicEvent):
+    """
+    :param int population: The ID of the population to activate.
```

Comment:
The attribute docstring gets associated with the attribute itself, see the Population class, e.g.

Reply:
Ok, I'll try to clean this all up as well

Reply:
I'm not sure if how I changed it is what you had in mind. Not really familiar with this way of adding doc to the attribute itself, so let me know if I'm off here.